data/json/recipes/recipe_electronics.json

```diff
@@ -364,11 +364,25 @@
   "components": [
     [ [ "amplifier", 1 ] ],
     [ [ "cable", 5 ] ],
-    [ [ "battery", 15 ] ],
     [ [ "e_scrap", 1 ] ]
   ]
 },{
   "type" : "recipe",
+  "result": "lightstrip_inactive",
+  "category": "CC_ELECTRONIC",
+  "subcategory": "CSC_ELECTRONIC_LIGHTING",
+  "skill_used": "electronics",
+  "difficulty": 0,
+  "time": 10000,
+  "reversible": false,
+  "autolearn": false,
+  "book_learn": [[ "manual_electronics", 1 ] , [ "mag_electronics", 1 ]],
```

Comment:
`"book_learn"` is redundant for 0 difficulty recipes, since those are taught at start to all characters.
`"reversible" : false` is always redundant and should be removed.
`"autolearn" : false` is always redundant and should either be changed to `true` or removed.